client/js/verification-portal/components/time.js

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
+import injectSheet from 'react-jss'
+import { connect } from 'react-redux'
+import { change } from 'redux-form'
+
+let i = 0
```

Comment:
Yes, we can, but defining the variable in every function call is memory consuming, and not a good practice.

Reply:
Hmm. Yeah technically it would consume more memory, but the amount is so small that I can't imagine it would make any difference in a noticeable way. In my opinion, it's much better to have more readable code than it is to address performance issues that don't cause actual noticeable differences (i.e the user actually notices the page is slower). Thoughts?